src/components/SummaryCard/SummaryCard.stories.js

```diff
@@ -146,6 +146,7 @@ storiesOf(patterns('SummaryCard'), module)
                 action('SummaryCardBatchAction onClick')(selectedSummaryCards)
               }
               renderIcon={Delete16}
+              tabIndex={getBatchActionProps().shouldShowBatchActions ? 0 : -1}
```

Comment:
@SimonFinney I'm curious if you know of a way *we* could add this on our end, without requiring users to add this in their implementation? 🤔 Otherwise any custom `SummaryCardBatchAction` they add will always have `tabIndex={0}` and therefore disrupt tab order when batch actions aren't even showing...

We may just need to add more documentation here.

Reply:
We'd need to investigate this further - I'm assuming we could add something to the `SummaryCardContainer` that handles this but I'm not entirely sure just yet 🤔